modules/apps/portal-language/portal-language-override-test/src/testIntegration/java/com/liferay/portal/language/override/service/test/PLOEntryLocalServiceTest.java

```diff
@@ -50,74 +50,76 @@ public class PLOEntryLocalServiceTest {
 		new LiferayIntegrationTestRule();
 
 	@Test
-	public void testAddPLOEntryNewKey() throws Exception {
-		String key = RandomTestUtil.randomString();
-
-		_assertTranslationValue(key, null);
+	public void testAddPLOEntry() throws Exception {
+		Locale defaultLocale = LocaleUtil.getDefault();
+
+		String languageId = LanguageUtil.getLanguageId(defaultLocale);
+
+		long companyId = TestPropsValues.getCompanyId();
+		long userId = TestPropsValues.getUserId();
+
+		try {
+			String key = StringPool.BLANK;
+
+			_ploEntryLocalService.addOrUpdatePLOEntry(
+				companyId, userId, key, languageId,
+				RandomTestUtil.randomString());
```

Comment:
@tototrinh 
Can you try to add the `Assert.fail();` at the end of the try block for the cases that throw the exception.